minimesos/src/test/java/com/containersol/minimesos/HelloWorldContainer.java

```diff
@@ -28,8 +29,15 @@ protected void pullImage() {
 
     @Override
     protected CreateContainerCmd dockerCommand() {
-        return dockerClient.createContainerCmd(HELLO_WORLD_IMAGE).withPrivileged(true)
-                .withName( getName() );
+        ExposedPort exposedPort = ExposedPort.tcp(SERVICE_PORT);
+        Ports portBindings = new Ports();
+        portBindings.bind(exposedPort, Ports.Binding(SERVICE_PORT));
+        return dockerClient.createContainerCmd(HELLO_WORLD_IMAGE)
+                .withEnv(String.format("SERVICE_%d_NAME=%s", SERVICE_PORT, SERVICE_NAME))
+                .withPrivileged(true)
+                .withName(getName())
+                .withPortBindings(portBindings)
+                .withExposedPorts(exposedPort);
     }
 
```

Comment:
Will a container be registered if it does not have a SERVICE_NAME env? For instance a framework executor?